enterprise/server/remote_execution/dirtools/dirtools.go

```diff
@@ -31,9 +31,7 @@ import (
 
 const gRPCMaxSize = int64(4000000)
 
-var (
-	enableDownloadCompresssion = flag.Bool("cache.client.enable_download_compression", false, "If true, enable compression of downloads from remote caches")
-)
+var enableDownloadCompresssion = flag.Bool("cache.client.enable_download_compression", false, "If true, enable compression of downloads from remote caches")
```

Comment:
can you revert the noisy diffs to `var` and `0111 -> 0o111` etc (I guess they are caused by gofumpt vs gofmt or something?)